src/Compilers/Core/Portable/Text/SourceHashAlgorithm.cs

```diff
@@ -22,4 +22,11 @@ public enum SourceHashAlgorithm
         /// </summary>
         Sha256 = 2,
     }
+
+    internal static class SourceHashAlgorithmUtils
+    {
+#pragma warning disable CA1802 // Use literals where appropriate
+        public static readonly SourceHashAlgorithm DefaultHashAlgorithm = SourceHashAlgorithm.Sha256;
```

Comment:
What is the reason for not making this a constant? All the binaries which have IVT access to this assembly are required to ship as a unit so there is no risk we'd have a stale constant in one of them. Having it be a constant though means we could use it as a default value in an optional parameter.